website/content/docs/agent/options.mdx

```diff
@@ -460,7 +460,10 @@ The options below are all specified on the command-line.
   "trace", "debug", "info", "warn", and "err". You can always connect to an agent
   via [`consul monitor`](/commands/monitor) and use any log level. Also,
   the log level can be changed during a config reload.
-
+- `-auto-reload-config` ((#\_auto_reload_config)) - This flag set Consul to automatically reload
+  [Reloadable Configuration](#reloadable-configuration) when configuration files change.
+  Consul will also watch certificate and key files set in `cert_file` and `key_file` and reload the configuration
+  if updated.
```

Comment:
```suggestion
- `-auto-reload-config` ((#\_auto_reload_config)) - This flag enables Consul to automatically
  [reload configuration files](#reloadable-configuration) when they have changed.
  Consul will also watch certificate and key files specified with the `cert_file` and `key_file` parameters and reload the configuration if updated.
```
Deemphasized "Reloadable Configuration" -- As far as I know, it's not a branded entity. I also tried to be more precise in the second sentence; please ignore if you don't feel that it is an improvement.